test/Instances/Attribute.spec.lua

```diff
@@ -0,0 +1,63 @@
+local Package = game:GetService("ReplicatedStorage").Fusion
+
+local New = require(Package.Instances.New)
+local Attribute = require(Package.Instances.Attribute)
+local Value = require(Package.State.Value)
+
+return function()
+    it("should create attributes (constant)", function()
+        local child = New "Folder" {
+            [Attribute "Foo"] = "Bar"
+        }
+        expect(child:GetAttribute("Foo")).to.equal("Bar")
+    end)
+
+    it("should create attributes (state)", function()
+        local attributeValue = Value("Bar")
+	    local child = New "Folder" {
```

Comment:
There is still some mixed indentation in this file, specifically line 17. Most of the file uses 4 spaces, but line 17 has one tab and then 4 spaces.

Reply:
I'm honestly not sure what's happening with VSC and it's indenting. I've never pressed space to tab at all :( I'll have a look through all the files and see what I can do!

Reply:
I've tried to fix the indenting again but I've got no idea whats going on at all :(

Reply:
@Dionysusnu 
![image](https://user-images.githubusercontent.com/91935721/213951376-7e7fe770-badf-45dd-807a-d94646201c13.png)
The above is what my code looks like, any reason why you think my indenting is broken?

Reply:
Because it's still using mixed tabs and spaces. (like in test/Instances/AttributeOut.spec.lua, line 22)
In Visual Studio Code, you probably have your tabs configured to show up as the same width as 4 spaces, but not everyone has this. Some people prefer 2 spaces or 8 spaces to represent a tab. It's important for that compatibility that files use either all tabs or all spaces.

Reply:
For what it's worth, I prefer tabs here so that different editors can flexibly show the indentation as per the user's preference, and also because screen readers can stumble with space-indented files (learned this from Rich Harris, will need to dig up the source though). Tab indentation is generally only a problem if lines of code are manually aligned, e.g.:

```Lua
local foo       = "longest"
local fooBar    =  "longer"
local fooBarBaz =    "long"
```

However, since Fusion does not do this, it is not a problem. As such, it is preferred to use tabs here.

Reply:
Hmm, well I'll be away for a few days so when I come back I'll try my best to fix this indenting problem!